app/views/users/show.html.erb

```diff
@@ -1,2 +1,25 @@
-<h1>This will display the data for an specific user
-</h1>
\ No newline at end of file
+<div class="profile">
+    <%= render partial: 'user_card', locals: { user: @user} %>
+    <div class="bio">
+        <h3>Bio</h3>
+        <p class="b-text"><%= @user.bio %></p>
+    </div>
+    <div class="section-post-container">
+        <ul class="posts">
+            <% @user.posts.each_with_index do |post, i| %>
+                <li>
+                  <a href="<%=user_post_path(@user,post)%>">
+                   <%= render partial: 'posts/post_card', locals: { post: post, post_number:i+1 } %>
+                  </a>
+                </li>
+            <% end %>
+        </ul>
+        <%# next button will redirect to see all post for current user %>
```

Comment:
- You have done great so far :+1: But there are a few requirements you need to meet before moving to the next milestone. :+1: :100: 

- [ ] In this user show page, after showing the user's details, you are meant to show only the users `three most recent posts` here. You are to do this using the `last_three_posts` method in your `user model`. I'll suggest you take this into consideration and update this. :smile: :wave:

Reply:
@V-Blaze Thanks for pointing that out! I'll make the correction!